pages/common/nms.md

```diff
@@ -0,0 +1,20 @@
+# nms
+
+> A cool (piped) text decryption effect.
+> More information: <https://github.com/bartobri/no-more-secrets>.
+
+- Decrypt "Hello, World!" after a keystroke:
+
+`echo "Hello, World! | nms"`
+
+- Decrypt the output of `ls -la` automatically:
+
+`ls -la | nms -a`
+
+- Decrypt the content of `secret_message.txt` automatically with red output:
```

Comment:
We try to be generic when possible, using simply 'a file' when it will work just as well as an actual file name.
```suggestion
- Decrypt the content of a file automatically with a custom output color:
```